server/src/endpoints/northstar/users.js

```diff
@@ -16,6 +16,16 @@ class NorthstarEndpointUsers extends OAuthEndpoint {
    * @return {Promise}
    */
   get(id, query) { return this.executeGet(`${this.url}/${id}`, query); }
+  /**
+   * @param  {String} email
+   * @return {Promise}
+   */
+  getByEmail(email) { return this.executeGet(`${this.url}/email/${email}`); }
```

Comment:
Will this send a request to `/users/email/:email`? I think we might need to adjust the `this.url` above!

Reply:
Eek, great catch! 💣 I forgot to update the `DS_NORTHSTAR_API_BASEURI` config variable over in  Gambit Slack, where I've been [testing this](https://github.com/DoSomething/gambit-slack/pull/53) via `npm link`. Will update